static/styles/zulip.scss

```diff
@@ -485,6 +485,7 @@ strong {
 .tooltip .tooltip-inner {
     background-color: hsla(0, 0%, 7%, 0.8);
     padding: 3px 5px;
+    font-size: 14px;
```

Comment:
What was the original size of the tooltips? You should probably add that to the commit message since the changes don't make it very clear.